addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -499,6 +499,299 @@ registry.slider = publicWidget.Widget.extend({
+    ],
+    disabledInEditableMode: false,
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    start: function () {
+        this._startVideo();
+        return this._super.apply(this, arguments);
+    },
+
+    /**
+     * @override
+     */
+    destroy: function () {
+        this._super.apply(this, arguments);
+        this._stopVideo();
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * When user provides video this method will update the video type according 
+     * to the input.
+     *
+     * @private
+     */
+    _updateVideoType: function () {
+        var regexUrl = '((?:https?:)?//([^\\s\'"<>\\\\/]+)/([^\\s\'"<>\\\\]+))';
+        var match = this.src.match(new RegExp('\\ssrc=[\'"]?' + regexUrl));
+        match = match || this.src.match(new RegExp('^\\s*' + regexUrl));
+        if (!match) {
+            this.videoType = "image";
+            this.src = "";
+            return;
+        }
+
+        var url = match[1];
+        var domain = match[2];
+        var path = match[3];
+
+        match = undefined;
+
+        var servicesPrefix = {
+            youtube: 'https://youtu.be/',
+            vimeo: 'https://vimeo.com/',
+            dailymotion: 'http://dai.ly/',
+        };
+
+        if (/\.youtube(-nocookie)?\./.test(domain)) {
+            this.videoType = 'Youtube';
+            match = path.match(/^(?:embed\/|watch\?v=)?([^\/?&#]+)/i);
+        } else if (domain === "youtu.be") {
+            this.videoType = 'Youtube';
+            match = path.match(/^([^\/?&#]+)/);
+        } else if (_.str.include(domain, "vimeo.")) {
+            this.videoType = 'vimeo';
+            match = path.match(/^(?:video\/)?([^?&#]+)/i);
+        } else if (_.str.include(domain, ".dailymotion.")) {
+            this.videoType = "dailymotion";
+            match = path.match(/(?:embed\/)?(?:video\/)?([^\/?&#_]+)/i);
+        } else if (domain === "dai.ly") {
+            this.videoType = "dailymotion";
+            match = path.match(/^([^\/?&#]+)/);
+        }
+
+        if (match) {
+            this.src = servicesPrefix[this.videoType] + match[1];
+        } else if (!/\ssrc=/.test(this.src)) {
+            this.src = url;
+            this.videoType = 'html5';
+        } else {
+            this.videoType = 'other';
+        }
+
+        this.$target.data('video-type', this.videoType);
+    },
+
+    /**
+     * When user provides video this method will be called first.
+     *
+     * @private
+     */
+    _startVideo: function () {
+        var self = this;
+        var videoUrl;
+
+        // mobile phones are not supported, just show the background image.
+        if (jQuery.browser.mobile) { return; }
```

Comment:
Can't we remove this now? This was a goal of the task to make it work in mobile :)